doc/how-to/db/crud.rst

```diff
@@ -194,367 +95,328 @@ INSERT
 DELETE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``delete`` accepts a full key of any unique index.
-
-``space:delete`` is an alias for "delete by primary key".
+:ref:`space_object.delete <box_space-delete>` allows you to delete a tuple identified by the primary key.
+You can also use :ref:`index_object.delete <box_index-delete>` to delete a tuple by the specified unique index.
 
 .. code-block:: tarantoolsession
 
-    tarantool> -- Insert some test data --
-    tarantool> s:insert{3, 4, 5}
-    ---
-    - [3, 4, 5]
-    ...
-    tarantool> s:insert{6, 7, 8}
-    ---
-    - [6, 7, 8]
-    ...
-    tarantool> s:insert{9, 10, 11}
-    ---
-    - [9, 10, 11]
-    ...
-    tarantool> s:insert{12, 13, 14}
-    ---
-    - [12, 13, 14]
-    ...
-    tarantool> -- Nothing done here: no {4} key in pk index --
-    tarantool> s:delete{4}
+    -- Insert test data --
```

Comment:
> WDYT about splitting into parts? For example, move the description comments out of the code and make each of them an intro to a smaller snippet.

Yep, I thought about that too. But given that all these atomic snippets belong to a single terminal session, having a big code snippet makes sense.

Reply:
Generally I agree. 
But this exact page is an overview of basic functionality, which is supposed to be simple. It should attract new users.

If I (as a newcomer not familiar with Tarantool) wonder: "how do I delete an object" and see a snippet this huge, it would scare me: "Do I really need 80 lines to do this?"
In turn, If I see a section with small snippets, I understand that it provides several basic scenarios that I can use for different needs.